recipe/meta.yaml

```
@@ -21,27 +21,33 @@ requirements:
     - setuptools
```

https://github.com/enthought/traitsui/blob/8.0.0/pyproject.toml#L48

```suggestion
    - setuptools >=61
```